packages/optimizer/README.md

```diff
@@ -220,22 +221,26 @@ Minifies the generated HTML output and inlined CSS.
 - default: `true`
 - used by: [MinifyHtml](lib/transformers/MinifyHtml.js), [SeparateKeyframes](lib/transformers/SeparateKeyframes.js)
 
-#### `preloadHeroImage`
+#### `optimizeHeroImages`
 
 Enables hero image optimization. Hero images will either be auto detected or you can explicitly mark these by adding the `data-hero` attribute:
 
 ```
 <amp-img data-hero src="foo.jpg" ...>
 ```
 
-The maximum number of hero images that can be marked up using `data-hero` is `2`.
+The maximum number of hero images that can be marked up using `data-hero` is `2`. This number can be increased using the `maxHeroImageCount` parameter.
 
-If no `data-hero` attribute is present, AMP optimizer auto-detects hero images for `amp-img`, `amp-iframe`, `amp-video`, or `amp-video-iframe` and injects a `link rel=preload`. Image preload links will only be generated if there is none already existing. For `amp-img` elements, it will also server-side render the `img` element inside the `amp-img` element. This greatly improves image rendering performance and reduces the [largest contentful paint](https://web.dev/lcp/) (LCP) metric from [Core Web Vitals](https://web.dev/vitals/).
+Hero images are optimized by server-side rendering the `img` element inside the `amp-img` element. This greatly improves image rendering performance and reduces the [largest contentful paint](https://web.dev/lcp/) (LCP) metric from [Core Web Vitals](https://web.dev/vitals/).
```

Comment:
IANAL, but I think "This [can/may] greatly improve"/"normally reduces", etc is more accurate, no? or are we 100% certain of this

Reply:
I'm 100% certain, but might miss some corner cases.  Rephrased.